examples/platform/linux/CommissionableInit.cpp

```diff
@@ -99,6 +100,33 @@ CHIP_ERROR InitConfigurationManager(ConfigurationManagerImpl & configManager, Li
         configManager.StoreProductId(options.payload.productID);
     }
 
+    if (options.vendorName.HasValue())
+    {
+        const char * vendorName = options.vendorName.Value().c_str();
+        configManager.StoreVendorName(vendorName, strlen(vendorName));
```

Comment:
Why are we doing strlen instead of using the actual length from the value?  This applies to a bunch of these.

Reply:
fixed